### PR TITLE
Add new resource api templates for international401

### PR DESCRIPTION
**Pull request by AnjanaSenanayake**
Add new resource api templates for international401

Related Issue: https://github.com/wso2-enterprise/open-healthcare/issues/1402

### CODE REVIEW

File: r4/resource-apis/account/service.bal [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// Dissemination of any information or reproduction of any material contained
+// herein is strictly forbidden, unless permitted by WSO2 in accordance with
+// the WSO2 Software License available at: https://wso2.com/licenses/eula/3.2
+// For specific language governing the permissions and limitations under
+// this license, please see the license as well as any agreement you’ve
+// entered into with WSO2 governing the purchase of this software and any
+// associated services.
+//
+//
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/service.bal]: Shouldn't say 'DO NOT MODIFY'. Users should modify this service.

File: r4/resource-apis/account/api_config.bal [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+// Dissemination of any information or reproduction of any material contained
+// herein is strictly forbidden, unless permitted by WSO2 in accordance with
+// the WSO2 Software License available at: https://wso2.com/licenses/eula/3.2
+// For specific language governing the permissions and limitations under
+// this license, please see the license as well as any agreement you’ve
+// entered into with WSO2 governing the purchase of this software and any
+// associated services.
+//
+//
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/api_config.bal]: Shouldn't say 'DO NOT MODIFY'.

File: r4/resource-apis/account/Package.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+| FHIR version         | r4 |
+| Implementation Guide | [http://hl7.org/fhir](http://hl7.org/fhir) |
+| Profile URL          |[http://hl7.org/fhir/StructureDefinition/Account](http://hl7.org/fhir/StructureDefinition/Account)|
+
+### Dependency List
+
+| Module | Version |
+| --- | --- |
+|
+
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/Package.md]: Should this be empty? If so, can we totally remove this?

File: r4/resource-apis/account/Config.toml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# sourceSystem = "http://localhost:9595"
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/Config.toml]: Do we need this file? We don't use sourceSystem anymore and info logs will anyways be printed.

File: r4/resource-apis/account/Ballerina.toml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.templates.r4.account"
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/Ballerina.toml]: We decided to prefix health.fhir.templates.international401.RESOURCE ?

File: r4/resource-apis/account/service.bal (hunk 1 above)
Comment (AnjanaSenanayake) [reply r4/resource-apis/account/service.bal]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1

File: r4/resource-apis/account/api_config.bal (hunk 2 above)
Comment (AnjanaSenanayake) [reply r4/resource-apis/account/api_config.bal]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1

File: r4/resource-apis/account/Package.md (hunk 3 above)
Comment (AnjanaSenanayake) [reply r4/resource-apis/account/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1

File: r4/resource-apis/account/Config.toml (hunk 4 above)
Comment (AnjanaSenanayake) [reply r4/resource-apis/account/Config.toml]: Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1

File: r4/resource-apis/account/Ballerina.toml (hunk 5 above)
Comment (AnjanaSenanayake) [reply r4/resource-apis/account/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1

File: r4/resource-apis/account/Package.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- Now service will be invoked and returns an Operation Outcome, until the code template is implemented completely.
+
+## Adding a Custom Profile/Combination of Profiles
+
+- Add profile type to the aggregated resource type. Eg: `public type Account r4:Account|<Other_Account_Profile>;`.
+    - Add the new profile URL in `api_config.bal` file.
+    - Add as a string inside the `profiles` array.
+    - Eg: `profiles: ["http://hl7.org/fhir/StructureDefinition/Account", "new_profile_url"]`
+
+## Onboard as a Choreo project
```
Comment (ThishaniLucas) [inline r4/resource-apis/account/Package.md]: We can't directly onboard as a choreo project. we need the openapi and the endpoints yaml.

Comment (AnjanaSenanayake) [reply r4/resource-apis/account/Package.md]: As discussed offline, for now the instructions related to on-boarding on Choreo are removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/823ce4003ce19803b81239a8fd2bc79c2f1803e8 until the openapi generation for fhir service is supported.

**Review by ThishaniLucas — DISMISSED** [review]

**Comment by sameeragunarathne** [comment]
Let's remove the templates for resources like Bundle, OperationOutcome, Resource, DomainResource, CodeSystem, ValueSet etc. Those are not exposed as a API.

**Comment by AnjanaSenanayake** [comment]
> Let's remove the templates for resources like Bundle, OperationOutcome, Resource, DomainResource, CodeSystem, ValueSet etc. Those are not exposed as a API.

Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/9963be70f92cfeef15bd3382e6be3a6bbf81322c

**Comment by AnjanaSenanayake** [comment]
Additionally removed the APIs for Capability Statement (there is a separate API for metadata) and Binary(in similar to Bundle, does not require a API) as well in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/b1844ac78f534a622743062b6b1f27573287369e

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**